include/jemalloc/internal/jemalloc_internal_inlines_c.h

```diff
@@ -325,6 +325,8 @@ imalloc_fastpath(size_t size, void *(fallback_alloc)(size_t)) {
 	tcache_t *tcache = tsd_tcachep_get(tsd);
 	assert(tcache == tcache_get(tsd));
 	cache_bin_t *bin = &tcache->bins[ind];
+  /* Suppress spurious warning from static analysis */
+  assert(bin != NULL);
```

Comment:
nit: indent is off

Reply:
Sorry, this commit was written before I had fixed my editor settings in response to your other reviews.